components/Ecosystem/EcosystemTags.tsx

```diff
@@ -0,0 +1,183 @@
+function EcosystemTags({ tags }) {
+    // | "bridge"
+    // | "dashboard"
+    // | "defi"
+    // | "explorer"
+    // | "gaming"
+    // | "nft"
+    // | "oracle"
+    // | "wallet"
+    // | "zk"
```

Comment:
can we delete this?

also, instead of having a separate function component for each label, can we create a <GenericTag /> and pass in the label and color / glow property as props? there is too much repeated styles here. here's some code i generated with chatGPT:

```
// Generic Tag Component
function Tag({ color, text, additionalStyles }) {
    return (
        <div className={`bg-${color} text-white text-xs font-bold px-2 py-1 rounded-md mb-4 ${additionalStyles}`}>
            {text}
        </div>
    );
}

// Tag Data
const tagData = {
    bridge: { color: 'blue-500' },
    dashboard: { color: 'yellow-500' },
    defi: { color: 'red-500' },
    // ... other tags
    testnetA5: { color: '[#E81899]', additionalStyles: 'glow-pulse' },
    // ... more tags
};

function EcosystemTags({ tags }) {
    return (
        <div className="flex flex-wrap gap-2">
            {tags.map(tag => {
                const { color, additionalStyles = '' } = tagData[tag] || { color: 'neutral-500' };
                return <Tag key={tag} color={color} text={tag} additionalStyles={additionalStyles} />;
            })}
        </div>
    );
}

export { EcosystemTags };
```